internal/jet/projection.go

```diff
@@ -31,24 +31,30 @@ func (pl ProjectionList) serializeForProjection(statement StatementType, out *SQ
 }
 
 // As will create new projection list where each column is wrapped with a new table alias.
-// tableAlias should be in the form 'name' or 'name.*'.
+// tableAlias should be in the form 'name' or 'name.*', or it can be an empty string, which will remove existing table alias.
 // For instance: If projection list has a column 'Artist.Name', and tableAlias is 'Musician.*', returned projection list will
-// have a column wrapped in alias 'Musician.Name'.
+// have a column wrapped in alias 'Musician.Name'. If tableAlias is empty string, it removes existing table alias ('Artist.Name' becomes 'Name').
 func (pl ProjectionList) As(tableAlias string) ProjectionList {
-	tableAlias = strings.TrimRight(tableAlias, ".*")
+	tableAliasWithDot := ""
+	if tableAlias != "" {
+		tableAlias = strings.TrimRight(tableAlias, ".*")
+		tableAliasWithDot = tableAlias + "."
+	}
 
```

Comment:
I think we can extract this (L38-42) into separate method like `reAlias` and use it in `ColumnList.As`, as there is no reason to have different behavior on `ProjectionList` and `ColumnList`.